source/triggers/database-triggers.txt

```diff
@@ -635,9 +635,14 @@ Expression` field. App Services evaluates the match expression against the
 change event document and invokes the Trigger only if the expression evaluates
 to true for the given change event.
 
+The "match expression" is basically a JSON document that specifies the query
+conditions using the :manual:`MongoDB read query syntax
+</reference/operator/query/>`.
+
 We recommend only using match expressions when the volume of Trigger events
 measurably becomes a performance issue. Until then, receive all events and
-implement the logic to ignore certain events in the Trigger Function.
+decide what to do (if anything) with any given event in the Trigger function
```

Comment:
Not sure what "with any given event" means. I think it's referring to the list that follows, but it's not clear. Could we try:
> Until then, receive all events and handle them individually in the Trigger function.
?